web/src/pages/CreateRoom/CreateRoom.module.scss

```diff
@@ -0,0 +1,140 @@
+main.createRoom {
+	height: 100vh;
+	max-width: 30rem;
+	margin: auto;
+}
+
+section.createRoomForm {
+	display: flex;
+  flex-direction: column;
+
+	label {
+		margin: 2rem 0 0.5rem 0;
+	}
+
+	input {
+		padding: 1rem;
+		width: 100%;
+		height: 2rem;
+		border-radius: 0.5rem;
+	}
+
+	/* Generated input range styles*/
```

Comment:
Apparently styling a range input is not easy. So I ended up using a CSS generator to get it close to the design. Not all the way there but close. I figured we can tweak as we go. I could also be convince to move these styles and the range input into its own component if we think it will be cleaner